### ISSUE TITLE
Cannot use ScreenRead to announce the elements in the chart

### ISSUE THREAD

**Issue by DaoDaoNoCode** [FormidableLabs/victory#1741]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

In PR #1687  , you added accessibility demo. However, what we can do is only use Tab to focus on the content inside a chart. We still cannot use Screen Reader to announce the `aria-label` of the content.

### Reproduction

Take the VoiceOver on MacOS as an example. If you Tab between the contents of any chart, it will not read the `aria-label` of that content.

# Feature Requests

### Checklist

- [x] I've read through the [Docs](https://formidable.com/open-source/victory/docs) and [Guides](https://formidable.com/open-source/victory/guides) to make sure this functionality doesn't already exist

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### Description

The reason that we cannot use Screen Reader here is that the `role` of `<svg>` here is hardcoded as `role="img"` so that the Screen Reader cannot access the content in the chart. It's better to provide the users with an interface to define the `role` of the `<svg>` tag.

**Comment by boygirl** [comment]
@DaoDaoNoCode you're right! `role` should be a prop on `VictoryContainer`. I'll make that change

**Comment by boygirl** [comment]
@DaoDaoNoCode Once I release the change above (later today) you will be able to specify your own role like so:

```
<VictroyChart
  containerComponent={<VictoryContainer role="presentation" />}
  ...
```

It will also work with other Victory container components you might want to use, such as `VictoryVoronoiContainer` as these all extend from `VictoryContainer`

**Comment by DaoDaoNoCode** [comment]
@boygirl Thanks! That's exact what I want.

**Comment by boygirl** [comment]
Released in `victory@35.3.5`
